### DISCUSSION TITLE
[Q] What's difference?

### DISCUSSION THREAD

**Discussion by chatblanc-ciel** [pemistahl/lingua-rs#257] [Q&A]
I found [`lingua-rs`](https://github.com/pemistahl/lingua-rs), [`lingua-py`](https://github.com/pemistahl/lingua-py) in your repositories.

What is the difference between lingua-ra and lingua-py?

----------------------------------------------------------
P.S. 
First, I cannot find [lingua-py in PyPI](https://pypi.org/project/lingua-language-detector/).
So, I tried creating python binding of lingua-rs as lingua-py  a little while ago.
What do you think about this initiative?

**Comment by pemistahl** [comment]
Lingua-rs and Lingua-py are feature-identical. I'm planning to create my own Python bindings for the Rust implementation of Lingua. These bindings will eventually replace the Python implementation. So please do not put effort into this.

**Reply by chatblanc-ciel** [reply]
I think that is great!

However, just very unfortunate.
Just until I found your `lingua-py` I have created a [binding](https://github.com/chatblanc-ciel/lingua-py), although not complete.
And I have already registered with [PyPI](https://pypi.org/project/lingua-py/).

Can I work it with you?

**Reply by pemistahl** [reply]
No worries, It's all open source. But I'd like to ask you to please remove the logo from the project repository and to rename the project on GitHub and PyPI to something like _lingua-unofficial-python-bindings_. I want your bindings to not be confused with my original projects and I want to clarify that these bindings are not officially supported by me. Is that possible please? Thanks a lot in advance. By the way, I will come up with official Python bindings shortly which will be part of the lingua-rs repository.

**Reply by chatblanc-ciel** [reply]
Sorry, I'm late for reply and not good at English.
I want to ask some questions.

1. You want me to change few things.

- rename project
- remove logo

2. You are going to plan addition binding in `lingua-rs`. When will you do it on current plan?
3. I can't contribute it(lingua-rs or its own binding). Is this same as your opinion?

Please check which fact is right, tell me difference & shortage. 
Thank you for your consideration.

**Reply by pemistahl** [reply]
You are mostly right.

1. Yes, you have understood correctly. Please remove the logo and rename the project if possible.
2. I'm currently working on the bindings but it will take some time. Your bindings do not cover the entire API. My goal is to cover the entire API of the Python implementation of Lingua.
3. I don't want to include your bindings into lingua-rs as they are incomplete and do not provide the same API as lingua-py. So please do not put more work into it.

**Reply by chatblanc-ciel** [reply]
1. OK. I'm working on it now to rave reviews. It is expected to change by the end of the month.
2. Thanks for showing us the roadmap. My bindings are a work in progress and provide only minimal language determination capabilities. (Based on my request. However, I really wanted to expand a little more.)
3. I was troubled for several days because it didn't seem to be communicated well. I don't want to merge my bindings into lingua-rs, I want to help you create bindings from 0 to lingua-rs.
I think this would be good for both of us, for my own sake and to accelerate the realization of the formula.
This was created at DeepL, but if you have any questions, I would be happy to answer them.

**Reply by chatblanc-ciel** [reply]
Thanks again!
Sorry for the inconvenience.
